benchmark_runner/common/oc/oc.py

```diff
@@ -27,6 +27,7 @@ class OC(SSH):
     SHORT_TIMEOUT = 600
     # sleep time between checks is 5 sec
     SLEEP_TIME = 3
+    DISK_ID_PREFIX_LEN = 13
```

Comment:
Where does the prefix length come from?  Why can't you use the entire string?

Reply:
No need to specify entry string anymore. The parsing of every storage id scsi or wwn is the same.

Reply:
I don't understand why you need a fixed prefix length (or any kind of prefix) here.

Reply:
Put another way, why aren't you just using the string without interpretation?

Reply:
The main problem is that ODF is using **wwn-** prefix and LSO using **scsi-** prefix.
So I added this code to workaround the disk id type wwn\scsi prefix.

Reply:
Right, but other than the 'scsi-3` vs. `wwn-0x` (6 characters rather than 13), the names are identical.

What I don't understand is why you don't simply use the full name.  If you need to use wwn- names for ODF, just use the full wwn- name.

Reply:
To extract the unique id from the storage, need to cut 13 characters.
**scsi-3600062b**2070785c028ae89f244909b6c
**wwn-0x600062b**2070785c028ae89f244909b6c

I use 'scsi-' for LSO disk.
This should resolve the issue in the long term, even if they revert ODF back to using 'scsi-' instead of 'wwn-'

Reply:
ODF PV changed the disk format from **scsi-** to **wwn-** in OCP 4.14
So in order to find the free disk we should 
subtraction all_workers_disk[scsi- format] - odf_worker_disk[wwn- format] 
Thats the reason I remove the storage type in both.

Reply:
I understand you need to remove the `scsi-3` and `wwn-0x` prefixes, but the other 7 bytes are not part of the prefix, they are part of the ID.

Reply:
For example, on various of my systems I have disks named

```
wwn-0x5000c500ac5f71a3
scsi-35000c500ac5f71a3

wwn-0x6d094660593af20022eb5a6eb8838bae
scsi-36d094660593af20022eb5a6eb8838bae

wwn-0x55cd2e414f75a670
scsi-355cd2e414f75a670
```

Note that the lengths of these names differ.  You cannot make any assumptions beyond the first 6 bytes.

Reply:
So what you recommend to do ?

Reply:
Use the entire string except for the first 6 bytes.